### ISSUE TITLE
[Request] Batch support

### ISSUE THREAD

**Issue by kenshin088** [PS3Xploit/PS3xploit-resigner#15]
Possible to resign multiple games (sequentially) and output the signed rap files with different file names? Would be nice to drop all the pkgs in a folder, raps in another and just have the system run through each one. Thanks.

**Comment by Caio99BR** [comment]
TL;DR: Actually `resign_windows.bat` do this [see here](https://github.com/PS3Xploit/PS3xploit-resigner#usage)

The `resign_windows.bat` resign multiple PKGs (not Games Folders¹) sequentially and output all then to `output/pkgs/`. And for RAPs, it's better output only one RIF PKG (with all signed RAPs)  than separate RIF PKGs for each signed RAP with a custom name.

Basically this script do the work very well (and very fast compared to sign each RAP file manually, put all signed RAPs on USB and inject one-by-one with PETT).

---

¹ If you mean put Game Folders on `pkgs` and output be a pkg signed and ready-to-use, this is the wrong tool for it. You will need more than just it.
